app/models/record.rb

```diff
@@ -122,6 +131,10 @@ def parse_author(xml_item)
         ""
       end
     end
+
+    def remove_prefixed_date_in(text)
+      text.gsub(/^\d{1,2}.\d{1,2}.\d{2,4}:?\s*/, "")
```

Comment:
You forgot to escape the `.` character.
Like this, it will also match `2910111994` for example.

This is the right regex here:
```ruby
text.gsub(/^\d{1,2}\.\d{1,2}\.\d{2,4}:?\s*/, "")
```

Reply:
nice

Reply:
done with fa1dea5